_posts/2020-11-09-Basics-With-Bioawk.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  "Basics with Bioawk!"
+date:   2020-11-09 4:45:00 -0800
+categories: Bioawk
+---
+
+# Basics with ~Babish~ Bioawk!
```

Comment:
I think you may need two tildes on each side to have it render as a strikethrough
https://www.markdownguide.org/tools/jekyll/